pyproject.toml

```diff
@@ -1,3 +1,99 @@
+[tool.poetry]
+name = "baseplate"
+version = "2.7.0a1"
```

Comment:
Notable change: right now, the version is embedded here and not based on SCM tags, so we'll need to manually update this file when making new releases.

We may want to consider adopting https://github.com/mtkennerly/poetry-dynamic-versioning (haven't used it, but looks like what we want)? I'm inclined to wait and do it in a followup PR once we're sure all the other machinery is working.